client/planning-extension/src/extension.ts

```diff
@@ -13,6 +13,9 @@ import {AssignmentsList} from './assignments-overview';
 import {IPlanningExtensionConfigurationOptions} from './extension_configuration_options';
 import {AutopostIngestRuleEditor} from './ingest_rule_autopost/AutopostIngestRuleEditor';
 import {AutopostIngestRulePreview} from './ingest_rule_autopost/AutopostIngestRulePreview';
+import ng from 'superdesk-core/scripts/core/services/ng';
```

Comment:
Extensions must not import anything neither from core nor planning. Instead either use superdesk api or planning api via bridge.